spa-react/src/components/Message/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface IDate {
+  [key: string]: {
+    date: { seconds: number; nanoseconds: number };
+  };
+}
+
+export interface IMessage {
+  date: { seconds: number; nanoseconds: number };
+  id: string;
+  message: string;
+  senderEmail: string;
+  senderName: string;
+  receivers: [];
```

Comment:
rename

Reply:
we need to sort messages by whom they need to be received to . We can have chats with 2 and more users in future (for example). So we are the sender and receivers - who will receive our messages